src/Microsoft.DocAsCode.MarkdigEngine.Extensions/Xref/XrefInlineShortParser.cs

```diff
@@ -16,7 +16,8 @@ namespace Microsoft.DocAsCode.MarkdigEngine.Extensions
 
     class XrefInlineShortParser : InlineParser
     {
-        private const string Punctuation = ".,;:!?`~";
+        private const string ContinuableCharacters = ".,;:!?~";
+        private const string StopCharacters = @"""'<>";
```

Comment:
also add `[]|`?
`[]` defend for markdown links.
`|` defend for table cell.

Reply:
I see. Have update.